modules/ROOT/pages/runtime-patch-updates.adoc

```diff
@@ -1,9 +1,13 @@
 
-= Managing Mule Runtime Patch Updates
-
+= Managing Mule Runtime Patch Updates for Runtime Fabric
 
 The following provides important information about Mule runtime engine patch updates and the Mule runtime engine release channels for Anypoint Runtime Fabric.
 
+
+== Version Rewrite
+
+Version rewrite used to override the build tag specified in a deployment request if the build tag was out of support or invalid. Starting October 3, 2023, version rewrite does not override any specified tags for any version. However, if a Mule app is deployed through the API without a build tag, the latest build is auto-selected.
```

Comment:
I read this wrong the first two times. I think you are using "used to" in the sense of "in earlier versions", right? If so, I suggest "In earlier releases, version rewrite overrode ..."